glean-core/src/upload/mod.rs

```diff
@@ -377,7 +369,7 @@ mod test {
         }
 
         // Get the pending ping directory path
-        let pending_pings_dir = upload_manager.directory_manager.get_dir();
+        let pending_pings_dir = dir.path().join("pending_pings");
```

Comment:
Let's declare the directory name as a constant in `directory.rs` and use it here.